aten/src/ATen/native/quantized/cpu/qclamp.cpp

```diff
@@ -0,0 +1,37 @@
+#include <ATen/ATen.h>
+#include <ATen/NativeFunctions.h>
+#include <ATen/core/op_registration/op_registration.h>
+#include <ATen/native/TensorIterator.h>
+#include <ATen/native/cpu/Loops.h>
+#include <ATen/native/quantized/cpu/quantized_ops.h>
+#include <ATen/quantized/Quantizer.h>
+
+#include <algorithm>
+
+namespace at {
+namespace native {
+
+DEFINE_DISPATCH(qclamp_stub);
+
+namespace {
+Tensor quantized_clamp(const Tensor& qx, Scalar min, Scalar max) {
+  Tensor qy;
+  qclamp_stub(qx.device().type(), qx, min, max, qy);
+  return qy;
+}
+
+class QClamp final : public c10::OperatorKernel {
+ public:
+  Tensor operator()(Tensor qx, Scalar min, Scalar max) {
+    return quantized_clamp(qx, min, max);
+  }
+};
+
+static auto registry = c10::RegisterOperators().op(
+    "quantized::clamp(Tensor qx, Scalar min, Scalar max) -> Tensor qy",
+    c10::RegisterOperators::options().kernel<QClamp>(
+        TensorTypeId::QuantizedCPUTensorId));
```

Comment:
I think we actually want to switch to registering new ops via `native_functions.yaml`, however someone needs to add support for namespaces there (i.e `quantized::`), so this is OK for now

Reply:
Yes. The corresponding float operation is torch.clamp. So eventually this would be torch.quantized.clamp?